js/jsmind.js

```diff
@@ -2754,6 +2754,13 @@
             var evt = e || event;
             if (!this.opts.enable) { return true; }
             var kc = evt.keyCode;
+
+            //combine keyCode
+            var combine = ((e.ctrlKey || e.metaKey) || 0) * 1000;
```

Comment:
```suggestion
            kc += ((e.ctrlKey || e.metaKey) << 12) + (e.altKey << 11) + (e.shiftKey << 10);
```
这样就能支持 CTRL+ALT+SHIFT 这种组合了。